include/GILGen/Context.hpp

```diff
@@ -321,14 +321,19 @@ class Context {
     gil::CallInst *
     buildCall(gil::Value functionPtr, llvm::ArrayRef<gil::Value> args)
     {
-        types::PointerTy pointerType
-            = llvm::cast<types::PointerTy>(functionPtr.getType().getType());
-        types::FunctionTy *funcType
-            = llvm::cast<types::FunctionTy>(pointerType.getPointee());
+        types::FunctionTy *funcType = nullptr;
```

Comment:
never nullptr initialize something that is definitely initialized, it hides warnings and can hide bugs